### PR TITLE
Run CI on Ruby 2.3 as well

### PR DESCRIPTION
**Pull request by dlackty**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 language: ruby
+sudo: false
```
Comment (ljharb) [inline .travis.yml]: We might require sudo here - all the tests are failing. Can you try removing this line?

**Comment by ljharb** [comment]
Also, the travis build implies that 2.3 isn't supported yet.

**Comment by dlackty** [comment]
@ljharb All cleared now 😉

**Comment by ljharb** [comment]
Awesome, thanks!

**Pull request merged**